vector/src/main/java/im/vector/app/features/home/room/list/home/HomeFilteredRoomsController.kt

```diff
@@ -67,7 +76,14 @@ class HomeFilteredRoomsController @Inject constructor(
     }
 
     override fun buildItemModel(currentPosition: Int, item: RoomSummary?): EpoxyModel<*> {
-        item ?: return RoomSummaryItemPlaceHolder_().apply { id(currentPosition) }
-        return roomSummaryItemFactory.create(item, roomChangeMembershipStates.orEmpty(), emptySet(), RoomListDisplayMode.ROOMS, listener)
+        return if (item == null) {
+            val host = this
+            RoomSummaryItemPlaceHolder_().apply {
```

Comment:
Nit: `roomSummaryItemPlaceHolder{ }` could be used
```suggestion
            roomSummaryItemPlaceHolder {
```

Reply:
`roomSummaryItemPlaceHolder` adds model to collector, but here we need to return item instead, so we can't really use this inside `buildItemModel` method :(

Reply:
fair enough :)